benchmarks/src/main/scala/com/github/mrpowers/spark/fast/tests/MyBenchmark.scala

```diff
@@ -0,0 +1,41 @@
+package com.github.mrpowers.spark.fast.tests
+
+import org.apache.spark.sql.SparkSession
+import org.openjdk.jmh.annotations._
+import org.openjdk.jmh.infra.Blackhole
+
+import java.util.concurrent.TimeUnit
+import scala.util.Try
+
+private class MyBenchmark extends DataFrameComparer {
+  @Benchmark
+  @BenchmarkMode(Array(Mode.AverageTime, Mode.SingleShotTime))
+  @Fork(value = 2)
+  @Warmup(iterations = 10)
+  @Measurement(iterations = 10)
+  @OutputTimeUnit(TimeUnit.NANOSECONDS)
+  def assertApproximateDataFrameEqualityWithPrecision(blackHole: Blackhole): Boolean = {
+    val spark = SparkSession
```

Comment:
Should we also import core project test dependency to reuse some of the already defined test utility?